CONTRIBUTING.md

```diff
@@ -35,6 +35,15 @@ goreleaser --snapshot --skip-publish --rm-dist
 
 ## Conventions
 
+### Command usage
+
+Command usage descriptions (namely `cobra.Command.Use`) should follow the [docopt](http://docopt.org/) syntax.
+
+Additionally:
+- Optional arguments should be the first in the usage string.
+  - If there is more than one optional argument, you can use `[options]` to abbreviate.
```

Comment:
Why not make our lives easier and always use options for any optional flags? Unless we have an automated way to handle this, I think it's going to be difficult to maintain and error prone.